web/src/app/projects/view/view.component.html

```diff
@@ -6,6 +6,7 @@
     <span>{{ project.title }}</span>
     <span fxFlex></span>
     <span fxLayout="row" fxLayoutGap="20px" fxLayoutAlign="center center" *ngIf="isAdmin()">
+      <button mat-raised-button type="submit" color="accent" routerLink="monitors">Monitors</button>
       <button mat-raised-button type="submit" color="accent" (click)="addRepository()">Add Repository</button>
```

Comment:
I know this is not part of your PR, but when you make some edits for your PR, could you please rename `Add Repository` to just `Repositories`, thank you 🤓

Reply:
done